### ISSUE TITLE
Show recommendations based on preferences on account page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/alnap#175]

**Comment by angievanhegan** [comment]
@Maria-alnap I made a note of this in the meeting. I don't think we had it as in scope for this project though. Can you confirm? Thanks

**Comment by Maria-alnap** [comment]
@angievanhegan, you're right, the recommendations will only come up for resources search results, not in the account page

**Comment by angievanhegan** [comment]
@Maria-alnap Do you want us to give you a quote for how much it will cost for the recommendations to appear on the user account page (based on their preferences)?

**Comment by Maria-alnap** [comment]
@angievanhegan Aaah! That's what you meant, recommendations based on their preferences ... Yes, it would be good to cost this up!

**Comment by angievanhegan** [comment]
@russ-manta Over to you for costing. Thanks

**Comment by russ-manta** [comment]
@angievanhegan I think this will be 1 day of work ie £560

**Comment by angievanhegan** [comment]
Angie to find out if client wants to bring this into scope.

**Comment by angievanhegan** [comment]
At meeting on Friday 30 June Maria confirmed that this will come into scope.

**Comment by russ-manta** [comment]
@angievanhegan not sure why I moved this into review - do you know if this is still to be done post launch?

**Comment by angievanhegan** [comment]
I think it's out of scope. I'll move it.

**Comment by angievanhegan** [comment]
@Maria-alnap Can you confirm this one is out of scope please? I just want to be sure. Thanks.
